src/pages/Graph/SummaryPanelClusterBox.tsx

```diff
@@ -0,0 +1,245 @@
+import * as React from 'react';
+import { Tab, Tooltip, TooltipPosition, Badge } from '@patternfly/react-core';
+import { style } from 'typestyle';
+import _ from 'lodash';
+import { RateTableGrpc, RateTableHttp } from '../../components/SummaryPanel/RateTable';
+import { SummaryPanelPropType, NodeType } from '../../types/Graph';
+import { getAccumulatedTrafficRateGrpc, getAccumulatedTrafficRateHttp } from '../../utils/TrafficRate';
+import { summaryFont, summaryHeader, summaryBodyTabs } from './SummaryPanelCommon';
+import { CyNode } from '../../components/CytoscapeGraph/CytoscapeGraphUtils';
+import { KialiIcon } from 'config/KialiIcon';
+import SimpleTabs from 'components/Tab/SimpleTabs';
+import { PfColors } from '../../components/Pf/PfColors';
+
+type SummaryPanelClusterBoxState = {
+  clusterBox: any;
+};
+
+const defaultState: SummaryPanelClusterBoxState = {
+  clusterBox: null
+};
+
+const topologyStyle = style({
+  margin: '0 1em'
+});
+
+export default class SummaryPanelClusterBox extends React.Component<SummaryPanelPropType, SummaryPanelClusterBoxState> {
+  static readonly panelStyle = {
+    height: '100%',
+    margin: 0,
+    minWidth: '25em',
+    overflowY: 'auto' as 'auto',
+    backgroundColor: PfColors.White,
+    width: '25em'
+  };
+
+  constructor(props: SummaryPanelPropType) {
+    super(props);
+
+    this.state = { ...defaultState };
+  }
+
+  static getDerivedStateFromProps(props: SummaryPanelPropType, state: SummaryPanelClusterBoxState) {
+    // if the summaryTarget (i.e. graph) has changed, then init the state
+    return props.data.summaryTarget !== state.clusterBox ? { clusterBox: props.data.summaryTarget } : null;
+  }
+
+  render() {
+    const clusterBox = this.props.data.summaryTarget;
+    const boxed = clusterBox.descendants();
+    const cluster = clusterBox.data(CyNode.cluster);
+
+    const numSvc = boxed.filter(`node[nodeType = "${NodeType.SERVICE}"]`).size();
+    const numWorkloads = boxed.filter(`node[nodeType = "${NodeType.WORKLOAD}"]`).size();
+    const { numApps, numVersions } = this.countApps(boxed);
+    const numEdges = boxed.edges().size();
+    // when getting accumulated traffic rates don't count requests from injected service nodes
+    const nonServiceEdges = boxed.filter(`node[nodeType != "${NodeType.SERVICE}"][!isBox]`).edgesTo('*');
+    const totalRateGrpc = getAccumulatedTrafficRateGrpc(nonServiceEdges);
+    const totalRateHttp = getAccumulatedTrafficRateHttp(nonServiceEdges);
+    const incomingEdges = clusterBox.cy().nodes(`[${CyNode.cluster} != "${cluster}"]`).edgesTo(boxed);
```

Comment:
I guess the difference in how edges are resolved may be the issue with the data mismatches between Total and In/Outcoming.